app-modules/etno/src/Support/CitationFormatter.php

```
@@ -58,7 +58,13 @@ public static function format(
             $parts[] = "https://doi.org/{$doi}";
         }
 
-        return $parts ? \implode(' ', $parts) : null;
+        if (! $parts) {
+            return null;
+        }
+
+        $formatted = \implode(' ', $parts);
+
+        return Str::ucfirst($formatted);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Str' in method 'format'. (undefined)

(StaticAccess)